src/controllers/LoginController.php

```diff
@@ -318,6 +319,10 @@ private function initTeamSelection(string $authType): void
                 array($this->App->Request->request->getInt('team_id')),
                 $this->App->Request->request->getString('teaminit_firstname'),
                 $this->App->Request->request->getString('teaminit_lastname'),
+                null,
+                false,
+                true,
+                $this->App->Session->get('teaminit_orgid'),
```

Comment:
Use [named arguments](https://stitcher.io/blog/php-8-named-arguments) `orgid: $this->App->Session->get('teaminit_orgid')` to skip positional arguments with default values.

Reply:
used named argument instead